demisto_sdk/commands/common/hook_validations/readme.py

```diff
@@ -0,0 +1,48 @@
+import subprocess
+import os
+from demisto_sdk.commands.common.tools import print_error, print_warning
+
+
+class ReadMeValidator:
+    def __init__(self, file_path):
+        self.file_path = file_path
```

Comment:
@bakatzir lets check if we can avoid this field.

Reply:
There is no schema for md files in the sdk , as a result this class cant inherit from base_validator and this field can not be avoided.